docker-compose.yaml

```
@@ -0,0 +1,36 @@
+version: '3'
+
+# This docker-compose is for developer convenience, not for running in production.
+
+services:
+
+  postgres:
+    image: postgres:16.3
+    restart: always
+    container_name: postgres
```

```suggestion
    container_name: postgres_hive
```
I will also update spark postgres container to `postgres_spark` so that they won't conflict each other.

---

I wouldn't bother, there's no real reason to run both of these at the same time. This is just for testing the hive server works,

---

When I set things up in CI I'll use the same postgres server but a new db name

---

👍